### PR TITLE
Firefox metrics fix

### PR DESCRIPTION
**Pull request by mollyling**
- Adds logging that shows up in Firefox
- Substantially increases estimate of random bytes required.

### CODE REVIEW

File: src/crypto.js [hunk 1]
```diff
@@ -7,24 +7,40 @@ var buf, offset = 0;
 var fcrypto = freedom['core.crypto']();
 // https://developer.mozilla.org/en-US/docs/Web/API/RandomSource/getRandomValues
 var kMaxSingleBuffer = 65536;
+var logMessages = [];
+
+function logMessage(str) {
+  logMessages.push(str);
+}
```
Comment (willscott) [inline src/crypto.js]: semicolons on functions. Does this need to be in both files? could it use `freedom.log` calls?

File: src/metric.js [hunk 2]
```diff
@@ -25,9 +42,17 @@ var getRandomness = function (callback) {
         }
     }
   }
-  return new Promise(function (resolve) {
-    console.log('asking to reload to ', 8 * 8 * bloombits + num_definitions);
-    crypt.refreshBuffer(8 * 8 * bloombits + num_definitions, resolve);
+  var num_desired_rand_bytes = 8 * 8 * bloombits * num_definitions;
```
Comment (willscott) [inline src/metric.js:28]: this seems to be the main change - multiplying by `num_definitions` right?

Comment (mollyling) [inline src/metric.js:28]: Yup. The rest is logging I added to debug this.

Cheers,
$ ls

**Comment by mollyling** [comment]
I added semicolons.   I'd rather not go through freedom.log.

**Comment by willscott** [comment]
That's fine. is there a way to not duplicate the logging code in the only 2
source files in the project? put them in a 3rd file and require them from
both, maybe?

**Comment by mollyling** [comment]
Debugging the changes to logging was a pain, so I stripped it out.  PTAL, thanks.

**Pull request merged**